tests/test_helpers.py

```diff
@@ -0,0 +1,33 @@
+from django.contrib.contenttypes.models import ContentType
+
+from djangocms_versioning.constants import DRAFT, PUBLISHED
+from djangocms_versioning.models import Version
+from filer.models import File, Folder
+
+from .base import BaseFilerVersioningTestCase
+from djangocms_versioning_filer import helpers
+
+
+class HelperTests(BaseFilerVersioningTestCase):
+
+    def test_check_folder_exists_in_folder(self):
+        self.assertFalse(helpers.check_folder_exists_in_folder(self.folder, 'olivertwist'))
+        subfolder = Folder(
```

Comment:
If we're not using factory boy on this repo (which I personally prefer as factories tend to be a lot easier to manage/maintain in my experience) then I suggest `Folder.objects.create(parent=self.folder, name='olivertwist')` rather than `Folder()` and then `subfolder.save()`.